app/services/post_alerter.rb

```diff
@@ -539,6 +546,49 @@ def expand_group_mentions(groups, post)
 
   end
 
+  def expand_here_mention(post, exclude_ids: nil)
+    exclude_ids ||= []
+    user_ids = []
+
+    # users that liked posts in topic
+    if user_ids.size < SiteSetting.max_here_mentioned
+      user_ids += PostAction
+        .where(post_id: Post.select(:id).where(topic_id: post.topic_id))
+        .where(post_action_type_id: PostActionType.types[:like])
+        .group(:user_id)
+        .order('COUNT(user_id) DESC')
+        .pluck(:user_id)
+      user_ids = (user_ids - exclude_ids).uniq
```

Comment:
Instead of `user_ids - exclude_ids`, I think we can just exclude the user id in the query above.